NEWS.md

```diff
@@ -1,3 +1,15 @@
+Version 0.8.0.5
+---------------------------------------------------------------------
+
+FEATURES / CHANGES
+
+- Improved various aspects of the documentation of `makeDisplay()`
+- Added overall package documentation.  See `help(trelliscope)`
+
+BUG FIXES
+
+- fix `makeDisplay()` to correctly launch trelliscope views on Windows
+
 Version 0.8.0.3
```

Comment:
Can you merge your features and bug fixes in with what's in the 0.8.0.3 section?  In other words, merge the version 0.8.0.5 section in to 0.8.0.3 and keep the header of Version 0.8.0.5.